packages/block-library/src/file/block.json

```diff
@@ -33,10 +33,17 @@
 			"type": "string",
 			"source": "html",
 			"selector": "a[download]"
+		},
+		"showInlineEmbed": {
```

Comment:
Should we call it "embed" or "preview"? I mean it's not that important but we could the same for like images, text .... any file that can be previewed. Something like "quick view" in MacOS

Reply:
`showPreview` or `displayPreview` perhaps?